packages/components/src/Icons/bin/wrapSVGUtils.ts

```diff
@@ -0,0 +1,39 @@
+// @ts-ignore: Redeclared module error
+const svgToComponentTitle = (fileName: string): string => {
+  const split = fileName.split("-")
+  return split
+    .map((word: string) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join("")
+    .replace(".icon", "Icon")
+}
+
+// eslint-disable-next-line no-template-curly-in-string
+const hrefReplacement = "href={`#${uniqueId}`}"
+
+// @ts-ignore: Redeclared module error
```

Comment:
Would it work if you don't use `module.exports` but instead export the function directly?
If it definitely doesn't, use `@ts-expect-error` instead :)

Reply:
Yeah it leads to this error sadly. Couldn't find a way around it.

```
export var svgToComponentTitle = function (fileName) {
^^^^^^

SyntaxError: Unexpected token 'export'
```

I might have to stick with `@ts-ignore` for this one, because the error only gets thrown in the tests, and not when executing the script. So `@ts-expect-error` throws an `Unused @ts-expect-error directive` error when executing the script